nipy/algorithms/clustering/tests/test_hierarchical_clustering.py

```diff
@@ -157,7 +159,7 @@ def ward_test_more(n=100, k=5, verbose=0):
     # Check that two implementations give the same result
     np.random.seed(0)
     X = randn(n,2)
-    X[:np.ceil(n/3)] += 5
+    X[:int(math.ceil(n / 3))] += 5
```

Comment:
@bthirion - I was thinking you meant this be floating point division, otherwise the `ceil` call doesn't make sense, because the integer division does an implicit `floor` on this positive number.  But you did mean integer division?  In which case we can discard the `ceil`.